diff --git a/nodequeue.module b/nodequeue.module
index f8f3bb4..cc5157f 100644
--- a/nodequeue.module
+++ b/nodequeue.module
@@ -506,7 +506,10 @@ function nodequeue_contextual_links_view_alter(&$element, $items) {
   );
 
   $elview = $element['#element']['#views_contextual_links_info']['views_ui']['view'];
-  $relationship = $elview->display_handler->handlers['argument']['name']->relationship;
+  $relationship = '';
+  if (!empty($elview->display_handler->handlers['argument']['name']->relationship)) {
+    $relationship = $elview->display_handler->handlers['argument']['name']->relationship;
+  }
   if (!empty($relationship) && $relationship == 'nodequeue_nodes_node') {
     $argument = $elview->display_handler->handlers['argument']['name']->argument;
   }
